### PR TITLE
add combine_pdfs function

### LINKED ISSUE #11: Helper function to combine pdfs
Use PyPDF2 and reticulate to combine files

### PR DESCRIPTION
**Pull request by chacalle**
## Describe changes

Add a helper function that uses the python `PyPDF2` package to quickly combine together pdfs into one file.

## What issues are related

Fixes #11 

### Packages Repositories

* [x] Have you read the [contributing guidelines](https://github.com/ihmeuw-demographics/packageTemplate/wiki#guide-to-r-package-development) for `ihmeuw-demographics` R packages?
* [x] Have you successfully run `devtools::check()` locally?
* [x] Have you updated or added function (and vignette if applicable) documentation? Did you update the 'man' and 'NAMESPACE' files with `devtools::document()`?
* [x] Have you added in tests for the changes included in the PR?
* [x] Do the changes follow the `ihmeuw-demographics` [code style](https://github.com/ihmeuw-demographics/packageTemplate/wiki/Code-style-guide)?
* [x] Do your changes need to be immediately included in a new build of [`docker-base`](https://github.com/ihmeuw-demographics/docker-base) or [`docker-internal`](https://github.com/ihmeuw-demographics/docker-internal)? If so follow directions in those repositories to rebuild and redeploy the images.

## Details of PR

We could just directly include the python function and name it `combine_pdfs` but I wasn't sure if that would make debugging harder for users. Versus doing the assertions in R with familiar functions.

### CODE REVIEW

File: tests/testthat/test-combine_pdfs.R [hunk 1]
```diff
@@ -0,0 +1,27 @@
+testthat::test_that("combine_pdfs works", {
+
+  testdir <- tempdir()
```
Comment (krpaulson) [inline tests/testthat/test-combine_pdfs.R:3]: Good use of `tempdir`

**Review by krpaulson — DISMISSED** [review]
This looks good. I agree on naming the python function different from the R wrapper. Maybe `combine_pdfs` and `combine_pdfs_py` though? Then as a convention we can have the names be `function_name` in R and `function_name_py` in python for situations like this.

**Comment by chacalle** [comment]
> This looks good. I agree on naming the python function different from the R wrapper. Maybe `combine_pdfs` and `combine_pdfs_py` though? Then as a convention we can have the names be `function_name` in R and `function_name_py` in python for situations like this.

That sounds like a good standard. Once this PR is done I'm going to add some information to the `packageTemplate` wiki about adding python functions to our packages

**Comment by chacalle** [comment]
I haven't quiet figured out the installation of the python packages within the github action workflow but going to create a separate issue and merge this in for now since it works for me locally and with our container

**Pull request merged**